sdk/search/azure-search/README.md

```diff
@@ -0,0 +1,211 @@
+# Azure cognitive search client library for Java
+
+This is the Java client library for [Azure Cognitive Search](https://docs.microsoft.com/en-us/rest/api/searchservice/).
+Azure Cognitive Search is a fully managed cloud search service that provides a rich search experience to custom applications.
+This library provides an easy (native) way for a Java developer to interact with the service to accomplish tasks like: create and manage indexes, load data, implement search features, execute queries, and handle results.
+
+[Source code][source_code] | [Package (Maven)][package] | [API reference documentation][api_documentation]
+| [Product documentation][search_docs] | [Samples][samples]
+
+## Getting started
+
+### Prerequisites
+
+-   Java Development Kit (JDK) with version 8 or above
+-   [Azure subscription][azure_subscription]
+-   An instance of [Cognitive search service][search]
+
+### Adding the package to your product
+
+[//]: # ({x-version-update-start;com.azure:azure-search;current})
+```
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-search</artifactId>
+    <version>1.0.0-beta.2</version>
+</dependency>
+```
+[//]: # ({x-version-update-end})
+
+## Using the library
+
+The APIs documented in this section provide access to operations on search data, such as index creation and population, document upload, and queries. When calling APIs, keep the following points in mind:
```

Comment:
@brjohnstmsft the setup is described in the Authenticate the client section of the README. The readme follows the same guidlines as all the rest of the SDKs.

Reply:
@navalev Since the setup is described in terms of the SDK elsewhere, can we remove this part? My concern is that it will be confusing for readers since it explains things in terms of calling the REST API directly.

Reply:
@brjohnstmsft , yes. removed. This can always be updated later